### DISCUSSION TITLE
Just wanted to make sure Tailwind and Alpine are available that "Tailwind" is not legible in dark mode on github.com

### DISCUSSION THREAD

**Discussion by justingolden21** [alpinejs/alpine#1055] [4. General]
Just wanted to make sure Tailwind and Alpine are available that "Tailwind" is not legible in dark mode on github.com

![image](https://user-images.githubusercontent.com/30274440/106372990-39892e80-632a-11eb-9517-518bcf486540.png)

Link to image currently used:
https://camo.githubusercontent.com/53b9876cd8e38928387c6824043b0e2772b15b1bfdb7f42d0864216abbf3dfe8/68747470733a2f2f7265666163746f72696e6775692e6e7963332e63646e2e6469676974616c6f6365616e7370616365732e636f6d2f7461696c77696e642d6c6f676f2e737667

Obviously making it white would be unreadable in light mode, and I don't know of a way to use different images in markdown depending on GitHub's theme, and I know a text outline would be ugly. So I suggest a white background, which, while it may not be ideal, will both keep the look in light theme the same, and improve dark theme to the text is readable. Just my two cents, obviously Tailwind should be the one to care more about this, and it's a minor thing since most devs will recognize the symbol anyway.

Thanks for listening to my feedback : )

**Comment by philwolstenholme** [comment]
This will only work with the system dark mode and not the manual toggle, but I wonder if this approach would work:

```
<picture>
  <source 
    srcset="dark.png" 
    media="(prefers-color-scheme: dark)">
  <img src="light.png">
</picture>
```

(Presuming Github allows `picture` elements and `source` elements of course)
